File: Module/Home/src/main/java/com/module/home/game/view/IGameView3.java
package com.module.home.game.view;

import com.module.home.model.GameKConfigModel;

public interface IGameView3 {
    void setGameConfig(GameKConfigModel gameKConfigModel);

    //显示红包运营为
    void showRedOperationView(GameKConfigModel.HomepagesitefirstBean homepagesitefirstBean);

    void hideRedOperationView();
}

File: Module/Home/src/main/java/com/module/home/game/view/IGameView3.kt
package com.module.home.game.view

import com.module.home.model.GameKConfigModel

interface IGameView3 {
    fun setGameConfig(gameKConfigModel: GameKConfigModel)

    //显示红包运营为
    fun showRedOperationView(homepagesitefirstBean: GameKConfigModel.HomepagesitefirstBean)

    fun hideRedOperationView()
}
